src/sage/rings/function_field/drinfeld_modules/charzero_drinfeld_module.py

```diff
@@ -0,0 +1,418 @@
+# sage.doctest: optional - sage.rings.finite_rings
+r"""
+Complex Drinfeld module
```

Comment:
Maybe "Drinfeld modules over rings of characteristic zero" instead?

Reply:
Yes thank you I forgot about this!